src/components/filterBox/FilterBox.tsx

```diff
@@ -53,18 +54,19 @@ export class FilterBox extends React.Component<IFilterBoxProps, any> {
 
   render() {
     let filterPlaceholder = this.props.filterPlaceholder || FILTER_PLACEHOLDER;
+    let filterBoxContainerClasses: string = `coveo-filter-container ${this.props.containerClass ? this.props.containerClass : ''}`;
```

Comment:
Personally I would go with an array like this:
```
let filterBoxContainerClasses = ['coveo-filter-container'].concat(this.props.containerClasses)
...
className={filterBoxContainerClasses.join(' ')}>
```
This way you could add more than one class